jussi/validators.py

```diff
@@ -251,6 +254,16 @@ def is_valid_get_block_response(
             request_block_num = params['block_num']
         else:
             raise ValueError(f'bad urn params from {jsonrpc_request}: {params} ')
+
+        if not isinstance(response, dict):
+            raise Exception(f'response was not a dict: {response}')
+        elif 'error' in response:
```

Comment:
already handled here with call to is_valid_non_error_single_jsonrpc_response here https://github.com/steemit/jussi/blob/master/jussi/validators.py#L243